controlplane/kubeadm/internal/machine_set.go

```diff
@@ -0,0 +1,120 @@
+/*
```

Comment:
:grimacing: It looks like I had forgotten to rename this file from `machine_set.go`, might be good to do here.

Reply:
whoops, yeah